build/benchmarks.matrix.azure.yml

```diff
@@ -15,24 +15,44 @@ groups:
       profiles:
       - aspnet-azure-lin
 
+    - name: Trends Database Azure Arm64 Linux
```

Comment:
Two jobs per group means it needs two queues at the top of the file. So we need to create another one in AzDo, might need to ask First Responders for that. `ASP.NET Benchmarks Azure Queue 2` for instance (not mentioning a name so we can use it for something else later). And create a new queue in the service bus service for it.

Reply:
Sure, I will use a generic name for now.